lib/SILOptimizer/Utils/InstOptUtils.cpp

```diff
@@ -1826,6 +1826,39 @@ void swift::endLifetimeAtLeakingBlocks(SILValue value,
       });
 }
 
+// Replace a store with an appropriate debug value.
+static void transferStoreDebugValue(DebugVarCarryingInst DefiningInst,
```

Comment:
Does `DefiningInst` imply that this will only work for
```
%1 = alloc_stack ..., var x
store %1 
```
and not
```
%1 = alloc_stack ...
debug_value, var x
store %1
```
?

Reply:
It is a DebugVarCarryingInst, so it will be called in both cases: in the first case DefiningInst will be the alloc_stack, in the second case the DefiningInst is the debug_value